framework/src/main/java/org/checkerframework/framework/type/SupertypeFinder.java

```diff
@@ -216,28 +216,50 @@
      */
```

Comment:
It wasn't changed in this pull request, but I'm curious why the supertypes of the enclosing types are included, but the supertypes of `type` are not included.  (Maybe "all type parameters of `type`" includes those in supertypes?  But in that case why wouldn't the wording also apply to the enclosing types.)

Reply:
(Add a Javadoc comment.)